cmd/token.go

```diff
@@ -0,0 +1,191 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/compute/metadata"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/google/go-tpm-tools/client"
+	"github.com/google/go-tpm-tools/launcher/agent"
+	"github.com/google/go-tpm-tools/launcher/spec"
+	"github.com/google/go-tpm-tools/launcher/verifier"
+	"github.com/google/go-tpm-tools/launcher/verifier/rest"
+	"github.com/google/go-tpm/legacy/tpm2"
+	"github.com/spf13/cobra"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+)
+
+var mdsClient *metadata.Client
+
+// If hardware technology needs a variable length teenonce then please modify the flags description
+var tokenCmd = &cobra.Command{
+	Use:   "token",
+	Short: "Attest and fetch an OIDC token from Google Attestation Verification Service.",
+	Long: `Gather attestation report and send it to Google Attestation Verification Service for an OIDC token.
+The OIDC token includes claims regarding the authentication of the user by the authorization server (Google IAM server) with the use of an OAuth client application(Google Cloud apps). Note that this command will only work on a GCE VM with confidential space image for now. And Confidential computing API needs to be enabled for your account to access Google Attestation Verification Service https://pantheon.corp.google.com/apis/api/confidentialcomputing.googleapis.com.
```

Comment:
If we verified EK cert is available on all CVMs, we can update the description here.